src/niswitch/metadata/config.py

```diff
@@ -43,6 +43,6 @@
             'python_name': 'channels'
         }
     ],
-    'session_class_description': 'An NI-SWITCH session to a National Instruments Switch Module',
+    'session_class_description': 'An NI-SWITCH session to a NI Switch Module',
```

Comment:
```suggestion
    'session_class_description': 'An NI-SWITCH session to an NI Switch Module',
```

Applies to other places too.